chapter02/sxg/src/main/java/com/github/sebin/chapter02/User.java

```diff
@@ -0,0 +1,36 @@
+package com.github.sebin.chapter02;
+
+public class User {
+	private int customerID;
+	private String username;
+	private String password;
+	private String fullname;
+	
+	/**
+	 * @param customerID
+	 * @param username
+	 * @param password
+	 * @param fullname
+	 */
+	public User(int customerID, String username, String password, String fullname) {
+		super();
```

Comment:
may be needed when we decided to change the super class from Object to something else..:)